core/templates/dev/head/pages/preferences/preferences.html

```diff
@@ -72,13 +72,13 @@
             <div class="col-lg-10 col-md-10 col-sm-10">
               <div class="checkbox" style="padding-top: 0;">
                 <label>
-                  <input type="radio" ng-model="defaultDashboard" value="DASHBOARD_TYPE_CREATOR" ng-change="saveDefaultDashboard(defaultDashboard)">
+                  <input type="radio" ng-model="defaultDashboard" value="<[DASHBOARD_TYPE_CREATOR]>" ng-change="saveDefaultDashboard(defaultDashboard)">
```

Comment:
Should we also have e2e tests to verify that the default dashboard is changed when the preferences are changed?